public/static/locales/eng/privacy-policy.md

```diff
@@ -2,108 +2,98 @@
+This statement deals in particular with the processing of personal data when individuals:
 
-Yfirlýsingin fjallar einkum um vinnslu persónuupplýsinga þegar einstaklingar:
+-   Visit the website, Samromur.is.
+-   Participate in the creation of the Samrómur database.
+-   Subscribe to Samrómur's mailing list.
 
-• heimsækja vefinn, [samrómur.is](https://samromur.is/).
+The University is committed to comply with applicable data protection legislation at all times. This Privacy Policy is based on Act no. 90/2018 on Data Protection and the Processing of Personal Data (“Data Protection Act”).
 
-• taka þátt í gerð gagnasafnsins Samrómur.
+# Privacy is important to RU
 
-• skrá sig á póstlista Samróms.
-
-HR vinnur persónuupplýsingar í samræmi við gildandi persónuverndarlöggjöf á Íslandi á hverjum tíma. Persónuverndaryfirlýsing þessi er byggð á lögum nr. 90/2018 um persónuvernd og vinnslu persónuupplýsinga („persónuverndarlög").
-
-# Persónuvernd er HR mikilvæg
-
-Öflug persónuvernd er HR kappsmál og leggjum við mikla áherslu á að virða réttindi einstaklinga og að öll meðferð persónuupplýsinga sé ávallt í samræmi við gildandi regluverk á hverjum tíma, sem og í samræmi við bestu venjur á sviði máltækni.
+Privacy protection is important to RU and we place great emphasis on respecting the rights of individuals and that all processing of personal information is always in accordance with current regulations at all times, as well as in accordance with best practices in the field of language technology.
 
 # Hvaða persónuupplýsingum safnar HR um þig og hver er tilgangurinn með söfnuninni?
 
-HR leggur áherslu á að vinna einungis þær persónuupplýsingar sem nauðsynlegar eru í samræmi við þann tilgang sem liggur að baki söfnun upplýsinganna. HR vinnur persónuupplýsingar ekki frekar í óskyldum tilgangi nema einstaklingi sé tilkynnt um slíkt og þá heimild sem vinnslan byggir á.
-HR safnar, eins og við á hverju sinni, einkum eftirfarandi persónuupplýsingum:
-
-• raddupptökum
-
-• lýðfræðilegum upplýsingum s.s. aldur, kyn og móðurmál.
-
-• samskiptaupplýsingum, s.s. tölvupósti
-
-• stafrænum fótsporum, s.s. nethegðun
-
-• upplýsingar tengdar samþykkis forsjáraðila fyrir barn s.s. kennitala barns og netfang forsjáraðila
+RU emphasizes the processing of only the personal information that is necessary in accordance with the purpose behind the collection of the information. RU does not further process personal information for unrelated purposes unless an individual is notified of this and the authorization on which the processing is based. RU collects, as applicable at any given time, the following personal information:
```

Comment:
```suggestion
RU emphasizes the processing of only the personal information that is necessary in accordance with the purpose behind the collection of the information. RU does not further process personal information for unrelated purposes unless an individual is notified of this and the authorization, on which the processing is based, permits it. 

RU collects, as applicable at any given time, the following personal information:
```
I changed the last sentence of the first paragraph a bit since it was ambiguous, should probably verify the change with ragnheidur.